docs/checkers/index.md

```diff
@@ -422,6 +422,26 @@ else:
     number_category = 'non-negative'
 ```
 
+(C0209)=
+
+### Consider using f string (C0209) [](#C0209)
+
+This error occurs when a string is formatted with `%` or `format()`.
```

Comment:
Add a second sentence: "The preferred way to include Python values in strings is with f-strings."